src/base.config.ts

```diff
@@ -115,8 +116,8 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 		}, {}),
 		node: { dgram: 'empty', net: 'empty', tls: 'empty', fs: 'empty' },
 		output: {
-			chunkFilename: `[name]-${packageJson.version}.js`,
-			filename: `[name]-${packageJson.version}.js`,
+			chunkFilename: `[name]/[name]-${packageJson.version}.js`,
```

Comment:
> I was having trouble getting file-loader to copy those svg/ttf assets into each widgets directory instead of just leaving them at the root.

To resolve this issue, shall we drop the folder and have all the elements built at the root, relative to the svg/ttf assets...? (ps don't forget to do the same for the css)

Reply:
Yeah, if we build everything to the root we don't have an issue. The only problem I can see with that is that you won't have a clear picture of what static assets go with what widgets, since they would no longer be bundled together.

Reply:
Yeah I think that's fine to be honest - as we won't know what assets are for what widget anyway so would have to blindly copy everything into every folder...